ethereum/eth/src/test/java/tech/pegasys/pantheon/ethereum/eth/manager/EthSchedulerTest.java

```diff
@@ -204,4 +207,19 @@ public void timeout_cancelsTaskWhenResultIsCancelled() {
     assertThat(task.isSucceeded()).isFalse();
     assertThat(task.isCancelled()).isTrue();
   }
+
+  @Test
+  public void scheduleTxWorkerTask_completesWhenScheduledTaskCompletes() {
```

Comment:
This name is misleading - there's no `Future` returned so nothing to complete.  It seems to just be testing that the mock executor immediately calls `run`.  As far as I can tell we're really just testing stubs so could just remove this test.

Unless I've missed something?

Reply:
Yeah this test is not relevant anymore.